### PR TITLE
Download and Save images locally

### PR DESCRIPTION
**Pull request by milensski**
This PR fixes issue #65 

I have implemented a manage command which is similar to the seed command for downloading one, staple, or all card images.
This was for analytic purposes, but it's good to have it there, just in case.

The logic for saving the cards is simple and if it's not a good approach please let me know.
It has one util function for downloading and saving a card image in our local media folder. After that, a serializer Mixin returns the local URL for the requested image in every serializer where the card image is needed.

### CODE REVIEW

File: backend/yugioh/utils.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    if not os.path.exists(image_file_name):
+
+        image_response = requests.get(external_image_url, headers=headers, stream=True)
+
+        if image_response.status_code == 200:
+
+            with open(image_file_name, 'wb') as handler:
+                for chunk in image_response.iter_content(chunk_size=8192):
+                    handler.write(chunk)
+        else:
```
Comment (julkascript) [inline backend/yugioh/utils.py:34]: It's better to handle else cases properly, or at least raise a warning/error (503).

File: backend/yugioh/utils.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    external_image_url = image_url
+
+    local_image_name = image_url.split('/')[-1]
+
+    media_path = os.path.join(settings.MEDIA_ROOT, 'yugioh_card_images')
+
+    image_file_name = os.path.join(media_path, local_image_name)
+
+    headers = {"User-Agent": "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0",
```
Comment (julkascript) [inline backend/yugioh/utils.py:17]: Are you sure you need the headers?

Comment (milensski) [reply backend/yugioh/utils.py]: > Are you sure you need the headers? What is their purpose in this case? Also, if yes, move them inside the if (before the request).

i was not sure, it actually works without them, and reformated the code.

Thanks.

**Comment by julkascript** [comment]
Fix conflicts

File: backend/yugioh/utils.py (hunk 1 above)
Comment (milensski) [reply backend/yugioh/utils.py:37]: > It's better to handle else cases properly, or at least raise a warning/error (503).

Now it should be ok -
I have made an exception to be handled properly, still returning the external image link while creating a log in the console for the exception.

**Review by dinocom33 — APPROVED** [review]

**Pull request merged**